INSTALLATION.md

```diff
@@ -1,3 +1,30 @@
+# Requirements and limitations
+
+## Target platform
+
+ This application is designed to run on IPhones with IOS. In addition, the application can be used on phones with Android OS.  But the display style of the app interface may differ from the usual, because the elements for creating the interface of this application designed with IOS-style. Also, correct display and use of the app on tablets is not guaranteed and depends on the support of their compatibility with Flutter-apps.
+
+## System requirements and limitations
+
+ For correct operation of applications on the ***IPhone*** you need:
+
+- IOS version 8.0 or higher
+
+- IPhone 4s or newer
+
+ At the moment, all elements of the application are displayed correctly for IPhones with a 4.7-inch display (1334x750 pixels, for example IPhone 6). App should be scaled without troubles to any higher resolution parameters. This information will be updated later.
+
+ For correct operation of applications on the ***Android*** you need:
```

Comment:
For correct operation of **the application** on ***Android*** **the following is required**: